### DISCUSSION TITLE
Migration supabase with snaplet

### DISCUSSION THREAD

**Discussion by devtayyab** [supabase/supabase#15142] [Questions]
I created a snapshot of my database using snaplet , and restore it to my staging enviorment. but when I try to use this database with my project it gives permission error.
![image](https://github.com/supabase/supabase/assets/57056497/761cdfce-3922-4cc3-8152-0a21bbfe5e45)

**Comment by luke-ransfield** [comment]
Does this help? [Supabase Discussion 5624](https://github.com/orgs/supabase/discussions/5624)
